encoding/README.md

```diff
@@ -2,6 +2,6 @@
 
 The `encoding` packages contain functions to encode and decode Cadence values to other formats.
 
-Currently, only the [JSON-Cadence](https://github.com/onflow/flow/blob/master/docs/json-cadence-spec.md) format is supported.
+Currently, only the [JSON-Cadence](https://github.com/onflow/cadence/blob/master/docs/json-cadence-spec.md) format is supported.
```

Comment:
This link should actually point to the rendered documentation page on the website:

```suggestion
Currently, only the [JSON-Cadence](https://docs.onflow.org/cadence/json-cadence-spec/) format is supported.
```

Reply:
Okey, I get the point let me update the code accordingly. Thanks for responding